flow/project.py

```diff
@@ -221,8 +230,11 @@ def never(cls, func):
 
     @classmethod
     def not_(cls, condition):
-        "Returns ``not condition(job)`` for the provided condition function."
-        return cls(lambda job: not condition(job),
+        "Returns ``not condition(job)`` for the provided condition function "
+        "for every job in the aggregate"
+        def _not(*jobs):
+            return not any(condition(job) for job in jobs)
```

Comment:
I am thinking we should change how this works. All other conditions that a user provides will now need to handle potentially multiple jobs or they will fail if called with a larger aggregate. The not here is designed to work with a standard condition, so the behavior should be the same.

```suggestion
            return not condition(*jobs)
```